### PR TITLE
Feature/mkd 17 route emp add

### PR DESCRIPTION
**Pull request by donovankg**

### CODE REVIEW

File: client/app/employee/addemployee/addemployee.controller.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  });
+    this.addEmp = function() {
+    if (this.newEmp.activate === 'true') {
+      this.newEmp.activate = true;
+    } else {
+      this.newEmp.activate = false;
+    }
+    this.newEmp.client = this.client;
+    this.newEmp.skill = this.skill;
+    this.newEmp.client_bill_pay = parseInt(this.newEmp.client_bill_pay);
```
Comment (lchapoy) [inline client/app/employee/addemployee/addemployee.controller.js]: Erase parseInt, is not neccesart and make it messier

File: client/app/employee/addemployee/addemployee.controller.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      types: workType
+    };
+  });
+    this.addEmp = function() {
+    if (this.newEmp.activate === 'true') {
+      this.newEmp.activate = true;
+    } else {
+      this.newEmp.activate = false;
+    }
+    this.newEmp.client = this.client;
```
Comment (lchapoy) [inline client/app/employee/addemployee/addemployee.controller.js]: You don't need any of this declarations

File: client/app/employee/employee.controller.js [hunk 3]
```diff
@@ -1,14 +1,36 @@
 'use strict';
 (function(Employee) {
 
+
   angular.module('maerkApp.employee')
-    .controller('EmployeeController', function(Employee) {
```
Comment (lchapoy) [inline client/app/employee/employee.controller.js]: Don't mix scope and  this, try to use only this

File: client/app/employee/employee.service.js [hunk 4]
```diff
@@ -1,26 +1,50 @@
...
-    return $resource('/api/employee/:id', {
-      id: '@_id'
-    }, {
-      getOne: {
-        method: 'get',
-        params: {
-          id: '@_id'
+  angular.module('maerkApp')
+    .factory('Empfactory', function($resource, EmpService) {
+      var resourceMethods = $resource('/api/employee/:id', {
```
Comment (lchapoy) [inline client/app/employee/employee.service.js]: Are you using this resourceMethods object

Comment (lchapoy) [inline client/app/employee/employee.service.js]: ????

**Comment by lchapoy** [comment]
I write some comments to change

File: client/app/employee/addemployee/addemployee.controller.js (hunk 2 above)
Comment (donovankg) [inline client/app/employee/addemployee/addemployee.controller.js]: removed this.

File: client/app/employee/addemployee/addemployee.controller.js (hunk 1 above)
Comment (donovankg) [inline client/app/employee/addemployee/addemployee.controller.js]: fixed this

File: client/app/employee/employee.service.js (hunk 4 above)
Comment (donovankg) [inline client/app/employee/employee.service.js]: this loads the employees, so yes its being used

File: client/app/employee/addemployee/addemployee.controller.js (hunk 1 above)
Comment (donovankg) [inline client/app/employee/addemployee/addemployee.controller.js]: removed code that wasn't needed

**Comment by donovankg** [comment]
I made some corrections

File: client/app/employee/employee.service.js (hunk 4 above)
Comment (lchapoy) [inline client/app/employee/employee.service.js]: You shouldn't have 2 $resource to the same endpoint

**Comment by lchapoy** [comment]
Changes

File: client/app/employee/employee.service.js (hunk 4 above)
Comment (donovankg) [inline client/app/employee/employee.service.js]: fixed

File: client/app/employee/employee.controller.js (hunk 3 above)
Comment (donovankg) [inline client/app/employee/employee.controller.js]: changes are finished

**Comment by lchapoy** [comment]
You can improve the design of you add dialog, a little padding between the sides of the dialog and the inputs should be good

**Comment by donovankg** [comment]
corrections made

**Pull request merged**